app/main.py

```
@@ -102,178 +50,64 @@ def get_contribution_stats():
-        user_query = st.text_input(
-            "Describe a plot! Let's see if we can find something that matches that."
-        ).strip()
-    with num_col:
-        num_recommendations = st.number_input(
-            "No. of results:",
-            min_value=1,
-            max_value=20,
-            value=config.model.top_k_recommendations,
+        items_html += contributor_item_template.substitute(
+            github=c["github"],
+            tooltip_text=tooltip_text,
```

Ruff (0.14.14)

[error] Undefined name `tooltip_text`

(F821)